tests/utils.go

```diff
@@ -2870,58 +2870,63 @@ func CheckForTextExpecter(vmi *v1.VirtualMachineInstance, expected []expect.Batc
 }
 
 func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter, virtClient kubecli.KubevirtClient, prompt string) error {
-	shouldConfigureIpv6 := func(vmi *v1.VirtualMachineInstance) bool {
-		shouldConfigureIpv6Batch := append([]expect.Batcher{
+	hasEth0Iface := func(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
You have not reason on why you re-introduced this one.

Please mention significant changes between pushes which are not part of the last review (so the reviewers will identify what changed).

Reply:
as we spoke, test 1776 doesn't have eth0, i removed but readded back the code, so we are just like the original, refactored

Reply:
I understand that `eth0` may be missing (which is a problem by itself), but you need to explain why it is not enough to just use `hasGlobalIPv6`.

Reply:
hasGlobalIPv6 might return false in case eth0 doesnt exists,
if eth0 doesnt exists, the function will try to config ipv6 on it and fail 
this is why we need to check that eth0 do exist and that it doesnt have ipv6 global, before we proceed configuration.

Reply:
OK, that is more clear now.

Can you paste here the test that did not have on the VM `eth0`?

Reply:
https://github.com/kubevirt/kubevirt/blob/82cbf4822c1708472e1dc2819705063125274a93/tests/vmi_networking_test.go#L584

test_id 1776